modules/affile/file-opener.h

```diff
@@ -28,6 +28,7 @@
 #include "compat/lfs.h"
 
 #include "file-perms.h"
+#include "lib/signal-slot-connector/signal-slot-connector.h"
```

Comment:
Let's just remove the `lib/` prefix, it should work too.

Reply:
Resolved